tests/foreman/api/test_docker.py

```diff
@@ -1058,19 +1058,32 @@ def test_update_internal_docker_compute_resource(self):
             compute_resource.update(['url']).url,
         )
 
-    @stubbed()
     def test_list_containers_internal_docker_compute_resource(self):
-        """@Test: Create a Docker-based Compute Resource in the
-        Satellite 6 instance then list its running containers.
+        """@Test: Create a Docker-based Compute Resource in the Satellite 6
+        instance then list its running containers.
 
-        @Assert: Compute Resource can be created, listed and existing
-        running instances can be listed.
+        @Assert: Compute Resource can be created and existing instances can be
+        listed.
 
         @Feature: Docker
 
-        @Status: Manual
-
         """
+        compute_resource = entities.DockerComputeResource(
+            organization=[self.org],
+            url=INTERNAL_DOCKER_URL,
+        ).create()
+        self.assertEqual(compute_resource.url, INTERNAL_DOCKER_URL)
+        self.assertFalse(entities.AbstractDockerContainer(
+            compute_resource=compute_resource).search())
```

Comment:
Can we assert that the length of the search results is zero? Type-casting a list of search results to a boolean works, but I find it a little non-obvious, and if the assertion fails, it would be more helpful to know how many results were returned.